packages/expo-av/src/Audio/Recording.ts

```diff
@@ -156,6 +168,7 @@ export type RecordingStatus = {
   isDoneRecording: boolean;
   durationMillis: number;
   metering?: number;
+  uri?: string | null;
```

Comment:
Thoughts?

It's added here because web has to report the uri when `stopAudioRecording()` is called, and this requires the least `if (Platform.OS === 'web' ...)` conditionals in Recording.ts
In all other cases it will be null, iOS/Android it'll be undefined.